core/src/main/scala/knobs/Parser.scala

```diff
@@ -38,22 +38,22 @@ object ConfigParser {
   }
 
   /** Top-level parser of configuration files */
-  lazy val topLevel: Parser[List[Directive]] = "configuration" |: {
-    directives << skipLWS << realEOF
-  }
+  lazy val topLevel: Parser[List[Directive]] = topLevel(withImport = true)
```

Comment:
Not a huge fan of shadowing the value name like this. Is there a reason this was done? Seems janky.